edgedb-init.sh

```diff
@@ -0,0 +1,269 @@
+#!/bin/bash
```

Comment:
Please don't forget to credit `rustup-init`, since this is derived from `rustup-init.sh`.

Reply:
I think something like what I suggested in https://github.com/edgedb/edgedb-cli/pull/75/files#r435523120 would be enough.

Reply:
Well, some things here are copied verbatim, including some comments, so it's not "inspired", it's literally based on (parts of) their code, which is totally OK, but we must be diligent in placing attribution, hence I'd put an explicit "Portions Copyright (c) 2016 The Rust Project Developers"

Reply:
Makes sense. I've amended my suggestion below.

Reply:
Sure. Just wasn't sure how to mention this.